hphp/runtime/ext/pdo_mysql/pdo_mysql.cpp

```diff
@@ -1281,7 +1285,11 @@ bool PDOMySqlStatement::getColumnMeta(int64_t colno, Array &ret) {
     ret.set(s_native_type, str);
   }
   ret.set(s_flags, flags);
+  ret.set(s_name, String(F->name, CopyString));
+  ret.set(s_org_name, String(F->org_name, CopyString));
   ret.set(s_table, String(F->table, CopyString));
+  ret.set(s_org_table, String(F->org_table, CopyString));
+  ret.set(s_length, F->length);
```

Comment:
This feels like a breaking change, is there any way that we could do this that would allow the default behavior to continue matching PHP? Also is there another bug filed against PHP about this? Has there been any discussion of it?

One possible option here would be to add an optional param to this function which indicate that we want these extra fields added. Another would be to only add them in HH mode. I'm not sure if we really need to bother with those though, maybe this isn't a big deal.